CHANGELOG.md

```diff
@@ -237,18 +264,27 @@ Fixed
 - Provides better stacktrace in `win_pkg` return. (#56955)
 - Fixed `reg.present` to respect `(Default)` REG_SZ value of an empty string. (#56959)
 - OpenStack driver can now attach to multiple networks, also now respects provided `conn`. (#56960)
+- Fix condition for ConfigTestCase.test_datetime_config_validation (#56968)
 - Fixed literal comparsion in `user` state. (#56972)
+- Additional fixes for using cron state with non-root Minion (#56973)
+- Master port of the missing commit from PR #50109 (#57010)
```

Comment:
I'd double-check this. If the PR is just fixing in-between release issues, it doesn't belong in the changelog. If this issue wasn't present in 3000.3 or earlier, then it's irrelevant to end users.